installer/debian/preinst

```diff
@@ -1,23 +1,27 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+set -e
 
 PKG_NAME=@PKG_NAME@
-VERSION="$2"
+INDEXER=algorand-indexer
 
-if test "$1" = install &&
-    dpkg-query --list 'algorand*' > /dev/null
+if [ "$1" != install ] || [ "$PKG_NAME" == $INDEXER ]
```

Comment:
PKG_NAME will never be algorand-indexer here though, right?

Reply:
It could be, if a user did `sudo apt-get install algorand-indexer`.

Reply:
Yes, but then they'd get the preinst built into the indexer package (and there isn't one over there)

Reply:
Oh, i see what you're saying, `algorand-indexer` isn't packaged with `preinst` so it will never be that value.  I had switched the order of operations and hadn't thought of that.  Thanks for catching that :)